src/pke/lib/scheme/ckksrns/ckksrns-fhe.cpp

```diff
@@ -276,9 +276,7 @@ Ciphertext<DCRTPoly> FHECKKSRNS::EvalBootstrap(ConstCiphertext<DCRTPoly> ciphert
         auto ctBootstrappedScaledDown = ctInitialBootstrap->Clone();
         auto bootstrappingSizeQ       = ctBootstrappedScaledDown->GetElements()[0].GetNumOfElements();
         if (bootstrappingSizeQ <= initSizeQ) {
-            OPENFHE_THROW(config_error, "Bootstrapping number of RNS moduli " + std::to_string(bootstrappingSizeQ) +
-                                            " must be greater than initial number of RNS moduli " +
-                                            std::to_string(initSizeQ));
+            return ciphertext->Clone();
```

Comment:
Out of curiosity why are we doing a clone here when we used to raise an exception? Is it to maintain parity with the behavior below?